kolibri/core/device/utils.py

```diff
@@ -75,3 +77,22 @@ def provision_device(**kwargs):
     device_settings, _ = DeviceSettings.objects.get_or_create(defaults=kwargs)
     device_settings.is_provisioned = True
     device_settings.save()
+
+
+def valid_app_key(app_key):
+    from .models import DeviceAppKey
```

Comment:
Does this file get imported somewhere pre-app-loading, or why do these imports need to be local?

Reply:
To prevent circular imports.